cli/dcoscli/data/help/node.txt

```diff
@@ -50,6 +57,14 @@ Options:
         Agent node with the provided ID.
     --user=<user>
         The SSH user, where the default user [default: core].
+    --list
+        List available snapshots.
+    --status
+        Print snapshot job status.
+    --cancel
+        Cancel a running snapshot job.
+    --download=<snapshot>
```

Comment:
can we gives users an option to download to a specific location and default to cwd or the directory we store the dcos-cli config?